### PR TITLE
Update useragent

### PR DESCRIPTION
**Pull request by DummySYSTEM**
update useragent base on chrome 67 x64

### CODE REVIEW

**Comment by eight04** [comment]
Why? Did any site refuse to accept current UA?

**Comment by DummySYSTEM** [comment]
base on my test seemh treat old useragent differently, the connection limit is higher when using old useragent, dunno if other site do the samething

File: comiccrawler/grabber.py [hunk 1]
```diff
@@ -15,7 +15,8 @@
 from .profile import get as profile
 
 default_header = {
-	"User-Agent": "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:34.0) Gecko/20100101 Firefox/34.0",
+	"User-Agent": "Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36
+",
```
Comment (eight04) [inline comiccrawler/grabber.py]: Syntax error.

Comment (DummySYSTEM) [reply comiccrawler/grabber.py]: when I edit in github web directly  it web into two line automatically, dunno how to solve it, sorry about that.

**Comment by DummySYSTEM** [comment]
It seems doens't make sense but consider seemh or other site may want to limit usage of downloader but can't ban the specific useragent, just like some web site will ask you update to newer browser if yours is outdated

**Comment by eight04** [comment]
I think it doesn't make sense because the current UA is a very outdated browser. Suppose there are only 1% users using Firefox 34, seemh would benefit almost nothing by limiting the bandwidth of 1% users. If they really want to limit the network bandwidth, they should apply the limitation to all users instead of 1% of them.

> some web site will ask you update to newer browser if yours is outdated

They don't check UA but testing the browser feature with JavaScript. UA might be fake so you can't trust it.

**Comment by DummySYSTEM** [comment]
I mean they target downloader like comiccrawler, as you said only 1% using the real firefox so limit client with known useragent will only have minor side effect. Some site does check UA although we know it can be fake easily, anyway, as you said it don't hurt update to newer useragent.

**Pull request merged**